### ISSUE TITLE
Color of text is sometimes not appropriate for block color

### ISSUE THREAD

**Issue by marohrdanz** [MD-Anderson-Bioinformatics/pathway-mapper#3]
There is code to select text color 'black' or 'white', depending on the color of the box. Sometimes it's not choosing the right thing, however:

![Screen Shot 2020-07-24 at 1 08 46 PM](https://user-images.githubusercontent.com/15746330/88421895-f2d0df80-cdae-11ea-8b9a-a046dd019760.png)

**Comment by marohrdanz** [comment]
This is addressed in 7c0095f, and once this is merged into the ngchm branch we can close this issue.
